src/DurableTask.Netherite/Scaling/LoadPublishWorker.cs

```diff
@@ -59,7 +66,7 @@ protected override async Task Process(IList<(uint, PartitionLoadInfo)> batch)
 
             try
             {
-                await Task.Delay(AggregatePublishInterval, this.cancelWait.Token).ConfigureAwait(false);
+                await Task.WhenAny(Task.Delay(AggregatePublishInterval), this.cancelWait.Task).ConfigureAwait(false);
             }
             catch (OperationCanceledException)
             {
```

Comment:
do we still need this catch statement? I think it's only for cancellation tokens.

Reply:
Good catch. I will remove the try/catch.